src/components/SelectionList/BaseListItem.tsx

```diff
@@ -25,6 +25,7 @@ function BaseListItem<TItem extends User | RadioItem>({
     onDismissError = () => {},
     rightHandSideComponent,
     keyForList,
+    shouldUseOnySubscriptAvatar = true,
```

Comment:
Typo

```suggestion
    shouldUseOnlySubscriptAvatar = true,
```

Could you give the case for this specific prop? I see that we pass `false` on the search page.